### DISCUSSION TITLE
双拼键盘方案

### DISCUSSION THREAD

**Discussion by wzgcn** [LufsX/rime#37] [General]
能否做一个双拼键盘方案呢？类似下面图片的样式。
<img width="400" alt="xhup" src="https://github.com/user-attachments/assets/42e8bee1-e6c5-466b-aeb4-241dc7449236">
这样有助于边记边练。

**Comment by LufsX** [comment]
键位图很多练习软件都有，参考一下就行

关于边记边练，个人比较支持在平常打字时强制使用双拼，使用键位图会造成依赖，不利于熟悉键位

另外，实在需要键位图参考参考话，手机上的小企鹅输入法一类可以直接套用「Google 拼音输入法双拼助记皮肤」使用
（你图示为小鹤音形，我的方案没有涉及到加形的操作，双拼记忆难度仅仅为三个特殊声母+韵母
